src/test/java/com/egov/business-services/tests/egfMasterChartOfAccountCreate.feature

```diff
@@ -0,0 +1,89 @@
+Feature: Chart of account Create
+Background:
+  * def jsUtils = read('classpath:jsUtils.js')
+  * def commonConstant = read('../../common-services/constants/genericConstants.yaml')
+  * def egfMasterChartOfAccountConstant = read('../../business-services/constants/egfMasterChartOfAccount.yaml')
+  # Required parameters to create acccount
+  * def glcode = ranInteger(6)
+  * def name = ranString(5) + "Bank"
+  * def isActiveForPosting = 'false'
+  * def functionRequired = 'false'
+  * def budgetCheckRequired = 'false'
+  * def classification = '1'
+  * def type = 'B'
```

Comment:
These values can be moved into `constant` file

Reply:
As per yatheendra, we should not keep these values in test feature instead of yaml file

Reply:
@Sarita1-eGov So, are you trying to say, these constants should be in the `test` feature not in the `constant.yaml` right? Okay, keep these under test itself then.

Reply:
I am not telling to keep it under feature file and as i said yatheendra told me to keep. I am using these values number of time in my script, so i feel better to define it in background.